### PR TITLE
Ma sprint 2

### PR DESCRIPTION
**Pull request by ma-sonr**
## Changes

- Removed references to Android/Java
- Re-ordered (CLI, Flutter, Go) 

## Checklist

- [ ] Unit tests
- [x] Documentation

## References *(optional)*

Fixes 
Connects 

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: src/components/SupportedClients.js [hunk 1]
```diff
@@ -7,9 +7,6 @@ function buildButton(type) {
     if (type === 'cli' || type === 'c') {
         return <span class="badge badge--secondary">CLI</span>
     }
-    if (type === 'android' || type === 'a') {
-        return <span class="badge badge--success">Android</span>
-    }
```
Comment (ntindle) [inline src/components/SupportedClients.js:12]: May be worth leaving this one?
```suggestion
    if (type === 'android' || type === 'a') {
        return <span class="badge badge--success">Android</span>
    }
```

File: docs/reference/motor/dart.md [hunk 2]
```diff
@@ -8,4 +8,4 @@ sidebar_position: 2
 ```
 ## motor-flutter API
 
-Matches the outline of [Android](/docs/reference/motor/java.md) (roughly). For more information, see the Motor Dart Docs on  [pub.dev](https://pub.dev/documentation/motor_flutter/latest/).
+For more information, see the Motor Dart Docs on  [pub.dev](https://pub.dev/documentation/motor_flutter/latest/).
```
Comment (ntindle) [inline docs/reference/motor/dart.md:11]: Motor Dart-> Motor Flutter. 

This won't work in dart, as it depends on flutter

File: docs/guides/setup.mdx [hunk 3]
```diff
@@ -8,39 +8,16 @@ import TabItem from "@theme/TabItem";
 import APIButton from "@site/src/components/APIButton";
 import SupportedClients from "@site/src/components/SupportedClients";
 
-The Motor SDK is a set of developer tools for building decentralized applications on the Sonr network. It currently supports Desktop, Android, and Flutter targets with plans to support iOS, React Native, and Web in the near future.
+The Motor SDK is a set of developer tools for building decentralized applications on the Sonr network. It currently supports Flutter (iOS, Android, MacOS) with plans to support Javascript, React Native, and Web in the near future.
```
Comment (ntindle) [inline docs/guides/setup.mdx:11]: Remove near from near future

**Review by ntindle — APPROVED** [review]

File: docs/guides/auth/login.mdx [hunk 4]
```diff
@@ -16,7 +16,7 @@ import {
   CreateBucketTerminalView,
 } from "@site/src/components/Terminal";
 
-<SupportedClients list={["a", "g", "f", "c"]} />
+<SupportedClients list={["g", "f", "c"]} />
```
Comment (ma-sonr) [inline docs/guides/auth/login.mdx:19]: @ntindle I removed as supported client

**Pull request merged**